scripts/SimpleQueries.sql

```diff
@@ -0,0 +1,16 @@
+-- Create a query to list out the event name and event data from the tblEvent table, with the most recent first.
+SELECT EventName, EventDetails, EventDate FROM dbo.tblEvent ORDER BY EventDate DESC;
+
+-- Create a query to list out the id number and name of the last 3 categories from the tblCategory table in alphabetical order
+SELECT * FROM dbo.tblCategory WHERE CategoryID > (SELECT COUNT(*) FROM dbo.tblCategory) - 3 ORDER BY CategoryName;
+
+-- Write a query to show the first 5 events (in date order) from the tblEvent table. 
+-- You should give the columns aliases (What and Details in this case); 
+-- Even though you're sorting by the event date, it shouldn't be included in your results. 
+SELECT TOP 5 EventName AS Event, EventDetails AS Details FROM dbo.tblEvent ORDER BY EventDate;
+
+-- Create a query which uses two separate SELECT statements to show the first and last 2 events in date order from the tblEvent table. 
+-- Redirect the output of this query to text, rather than to grid.
+SELECT TOP 2 EventID, EventName, EventDetails, EventDate FROM dbo.tblEvent
```

Comment:
Here it should get the first and last two by date
![image](https://github.com/TheCSharpAcademy/CodeReviews.SQL.SimpleQueries/assets/108711476/cdb10433-eda4-4860-8f36-cd5dc32306eb)

Reply:
Okay ))